### ISSUE TITLE
Meeting Agenda April 11, 2024

### ISSUE THREAD

**Issue by ghost** [WordPress/Learn#2369]
Training Team meetings start at [00:00 UTC weekly on Thursdays](https://make.wordpress.org/meetings/#training). Anyone is welcome to contribute to the meeting by commenting in Slack threads during the meeting and throughout the following days. Meeting notes will be published by the following Tuesday to also incorporate these asynchronous conversations.

_This week’s meeting will be hosted by @lada7042 ._

## 1. News

**1. [Meeting Note Takers](https://make.wordpress.org/training/handbook/about/team-roles/)**

- April 11 – [@Tapan](https://profiles.wordpress.org/voboghure/)
- April 18 – [@Arun Sharma](https://profiles.wordpress.org/dextorlobo/)
- April 25 – [@Sancastiza](https://profiles.wordpress.org/sancastiza/)
- May 2 – [@Jagir Bahesh](https://profiles.wordpress.org/jagirbahesh/) 

**2. Looking for feedback**

- A conversation was started in the last meeting. I want to give it more time if anyone has any other comments. https://wordpress.slack.com/archives/C02RW657Q/p1712191633797929
- Topic Summary- Members of the WordPress Performance Team have been working on an introduction to gathering WordPress performance data in the field. This is provided in [this Colab](https://colab.research.google.com/drive/1TmbJhSBuM-gTHA_E_W96pSjraQkFYpyi), as the Colab format is a great fit for this kind of content, since it allows providing BigQuery queries that can be run directly inside of it, and the results can be presented alongside the queries. Colab covers both potential core use-cases as well as plugin or theme use-cases.

**3. Looking for volunteers**
- Need help with reviewing lessons. If you know anyone who might want to contribute. [Reviewer application](https://make.wordpress.org/training/handbook/tutorials/reviewing-tutorial-presenter-applications/) Let us know. 

**4. Updates from last week's dev-squad triage session**

- The Training Team developer squad meets every other Thursday at 07:00 in [#meta-learn](https://wordpress.slack.com/archives/C01KC5VDQBC). You can learn more about the dev squad and what it does in [this handbook page](https://make.wordpress.org/training/handbook/training-team-how-to-guides/dev-squad-guidelines/).

**5. Other news**
I want to let new people here that the training team has a monthly newsletter and a monthly update. 

- The [Learn WordPress Newsletter – April 2024](https://learn.wordpress.org/learn-wordpress-newsletter-april-2024/) is now published -- check out all the new content we published last month!

- Our March team update is now available for review in the [Training Team Update – March 2024](https://make.wordpress.org/updates/2024/04/03/training-team-update-march-2024/) post -- this gives a rundown of our team goal progress, projects, and overall team health.

- Check out a new edition of [Contributor Spotlight](https://make.wordpress.org/training/2024/04/09/contributor-spotlight-cynthia-norman/)! @maya just published @cynthianorman 's Contributor Spotlight. Great to know more about your journey, Cynthia!

## 2. Come and Contribute

**[Content ready for review](https://github.com/orgs/WordPress/projects/33/views/17)**

- See [Guidelines for reviewing content](https://make.wordpress.org/training/handbook/training-team-how-to-guides/guidelines-for-reviewing-content-on-learn/) when reviewing content.
- We have 19 issues ready for review

**[Feedback awaiting validation](https://github.com/orgs/WordPress/projects/78/views/3)**

- See [Validating and Applying Content Feedback](https://make.wordpress.org/training/handbook/training-team-how-to-guides/how-we-use-github/validating-and-applying-content-feedback/) for step-by-step guidance on validating feedback.
- We have 23 issues that require validation

**[Topics awaiting vetting](https://github.com/orgs/WordPress/projects/105/views/1)**

- See [Vetting Topic Ideas](https://make.wordpress.org/training/handbook/training-team-how-to-guides/how-we-use-github/vetting-topic-ideas/) for step-by-step guidance on vetting topic ideas.
- We have 31 issue that requires vetting

**[Good first issues for developers](https://github.com/orgs/WordPress/projects/71/views/7)**

- See [Developing Learn WordPress](https://make.wordpress.org/training/handbook/training-team-how-to-guides/developing-learn-wordpress/) for instructions on contributing to the Learn WordPress code.
- We have 15 issues that need a look

**[Validated feedback awaiting fix](https://github.com/orgs/WordPress/projects/78/views/4)**

- Faculty Editors, see [Validating and Applying Content Feedback](https://make.wordpress.org/training/handbook/training-team-how-to-guides/how-we-use-github/validating-and-applying-content-feedback/) for step-by-step guidance on applying feedback that has been validated.
- We have 16 issues awaiting a fix

## 3. Contribution Acknowledgement

- Badges awarded: 
- Props: 

## 4. Project updates

- Learning Pathways
     - 👀 For content creators and reviewers. We could really use some help. 

## 5. Faculty Updates

- What have you been working on and how has it been going?
- Anything you’ve accomplished since the last meeting?
- Do you have any blockers?
- Can other Faculty or Training Team members help you in some way?

## 6. Open Discussions

If you have topics you’d like discussed in the meeting, please leave them as a comment on this issue.
- Sent in by @flexseth - I'd love to be able to link to timestamps in [WP.tv](http://wp.tv/) videos! Like when you're on YouTube and you click to share a video, it allows you to share the link "starting at X time" 🙂
---

You can see all meetings scheduled on [this meeting calendar](https://make.wordpress.org/meetings/#training). If you are new to the Training Team, then come walk through [our onboarding program](https://make.wordpress.org/training/handbook/getting-started/) to get to know the team and how we work. And if you have questions, feel free to reach out in [the #training Slack channel](https://wordpress.slack.com/archives/C02RW657Q) at any time.

**Comment by flexseth** [comment]
I'd love to be able to link to timestamps in [WP.tv](http://wp.tv/) videos! Like when you're on YouTube and you click to share a video, it allows you to share the link "starting at X time"  :slightly_smiling_face:

**Comment by ardianimaya** [comment]
Hi team! I won't be able to join this week's meeting as I'll be on Eid holiday until next week, but I have a couple of updates:

- I just published @cynthianorman 's Contributor Spotlight https://make.wordpress.org/training/2024/04/09/contributor-spotlight-cynthia-norman/ (great to know more about your journey, Cynthia!)
- I informed Wes and Ben that I cannot continue recording my script for [this tutorial](https://github.com/WordPress/Learn/issues/1727) anytime soon, and I'm open if another contributor would love to take it over (feel free to tweak and adapt my script as needed). So sorry for not being able to finish this issue :(

Have a great week ahead, folks!
